python/valis/routes/query.py

```diff
@@ -191,3 +191,13 @@ async def obs(self,
         """ Perform a search on carton or program """
 
         return list(get_targets_obs(release, obs, spectrograph))
+
+    @router.get('/mapper', summary='Perform a search for SDSS targets based on the mapper',
+                response_model=List[SDSSidStackedBase], dependencies=[Depends(get_pw_db)])
+    async def get_target_list_by_mapper(self, 
+                                        mapper: MapperName = Query(default=MapperName.MWM, description='Mapper name', example=MapperName.MWM),
+                                        page_number: int = Query(description='Page number of the returned items', gt=0, example=1),
+                                        items_per_page: int = Query(description='Number of items displayed in a page', gt=0, example=10)):
+        """ Return an ordered and paged list of targets based on the mapper."""
+        targets = get_paged_target_list_by_mapper(mapper, page_number, items_per_page)
```

Comment:
it's now recommended to move the `Query` into  an `Annotated` type when defining query or parameters, instead of setting them as the default.  Can you convert these to that syntax?  You can look at the usage in the other route endpoints for example.  Or from the main docs, https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#add-query-to-annotated-in-the-q-parameter.